addons/sale_crm/wizard/crm_opportunity_to_quotation.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class Opportunity2Quotation(models.TransientModel):
+
+    _name = 'crm.quotation.partner'
+    _description = 'Create new or use existing Customer on new Quotation'
+    _inherit = 'crm.partner.binding'
+
+    action = fields.Selection([
```

Comment:
So we redifine the field in order to remove the entry "Do not link to a customer" ? Don't we want to handle that use case ?

Reply:
A wise man said : "Nope, you need a customer on your quotation"

Reply:
That man is ... merveilleux !